test/tests-no-moment.js

```diff
@@ -42,23 +42,6 @@ define(function(require){
 		};
 	});
 
-	require('test/checkbox-test');
-	require('test/combobox-test');
 	require('test/datepicker-test');
```

Comment:
Thanks must have lost in merge manual merge conflicts. Will update in the morning.